sententia/src/main/java/nl/nfi/sententia/SententiaPlugin.java

```diff
@@ -170,10 +171,13 @@ public void domainObjectChanged(DomainObjectChangedEvent event) {
 		Iterator<DomainObjectChangeRecord> it = event.iterator();
 		while (it.hasNext()) {
 			DomainObjectChangeRecord evt = it.next();
-			if (evt.getEventType().getId() == ProgramEvent.SYMBOL_RENAMED.getId()) {				
+			// Check if we're dealing with a rename and the is applied to a function
```

Comment:
*and the is applied*? Should that be "that is applied"?

Reply:
Yes, it was a bit late and apparently my brain had turned to goo 🙃